target/linux/mvebu/base-files/etc/uci-defaults/04_mambafan

```diff
@@ -10,10 +10,20 @@ board=$(mvebu_board_name)
 case "$board" in
 armada-xp-linksys-mamba)
         # Set one execution of the fan script during every start (waiting for cron)
-        echo "/sbin/fan_ctrl.sh" > /etc/rc.local
+        if grep -q fan_ctrl.sh /etc/rc.local
+                then
+                        exit 0
+                else
+                        echo "/sbin/fan_ctrl.sh" > /etc/rc.local
```

Comment:
The thing with rc.local is that it comes by default with an ending "exit 0", so I can't "append" (>>) to rc.local as the files execution would have ended by the time anything after the "exit 0" comes. As well, I am sure you would not like the idea of me having to copy the contents to a temporary staging file to read out the contents of rc.local. Then I add my entry to the now empty rc.local, and then "append" the contents of the staged file back to rc.local, thus placing the fan execution at the top.

Instead I opted to just replace the file (but was only thinking of a clean install, not taking into account an "upgrade" scenario. I am open to suggestions of course if you think you have a cleaner way around this without me having to stage the contents to then write back.